### PR TITLE
Add Xamarin.Mac to net-standard support table

### PR DESCRIPTION
**Pull request by chamons**
# Title

Xamarin.Mac supports netstandard by leveraging mono just as Xamarin.iOS.

## Summary

Fixed https://github.com/dotnet/standard/issues/419

## Details

As the lead of Xamarin.Mac, I can claim support. :)

If it regresses file a bug at https://bugzilla.xamarin.com/index.cgi

### CODE REVIEW

**Review by mairaw — APPROVED** [review]
Thanks @chamons. If you're saying so, I believe you. :-)

**Comment by chamons** [comment]
Also fixed it here: https://github.com/dotnet/standard/pull/420

**Comment by ghuntley** [comment]
Thankyou!

**Review by terrajobst — APPROVED** [review]
LGTM, assuming Xamarin.Mac 3.0 implements .NET Standard 1.6 and 2.0 support is coming in the next version.

**Comment by chamons** [comment]
@terrajobst 3.0 implements netstd16 (along with 3.2, 3.4, etc). netstd20 was waiting on mono support and will land support same release as Xamarin.iOS.

**Pull request merged**
